content/people/dcallega.md

```diff
@@ -6,19 +6,21 @@ yearjoined: 2016
 portrait: dcallega.jpg
 badge: dcallega-badge.jpg
 interests:
-  - Distributed Machine Learning in Heterogeneous Computer Networks
-  - Reinforcement Learning and Semantic Maps
-  - Unmanned Aerial Vehicles
-  - Edge Computing
+  - distributed machine learning in heterogeneous computer networks
```

Comment:
Keep the very first letter capitalized, of course.
E.g.: “Reinforcement learning and semantic maps”.